### PR TITLE
lionLJX-2021-finalproject

### PR DESCRIPTION
**Pull request by lionLJX**
It is my first PR.

### CODE REVIEW

**Comment by lionLJX** [comment]
I need you review 
@MuZhou233

**Comment by lionLJX** [comment]
I think my homework mey be finished .please tell me what I can do and if I have some error ,please let me know!

**Comment by MuZhou233** [comment]
> I think my homework mey be finished .please tell me what I can do and if I have some error ,please let me know!

Pass the check action first

**Comment by lionLJX** [comment]
I make some changes and I need check action

**Comment by MuZhou233** [comment]
> I make some changes and I need check action

As you can see, your branch has conflicts so check action won't run

**Comment by lionLJX** [comment]
I solve my conflict ,please approve my PR,or tell me what I can do in the next step.

**Comment by lionLJX** [comment]
@MuZhou233

File: _2021-finalproject/README.md [hunk 1]
```diff
@@ -1,63 +1,63 @@
-# Final Project
```
Comment (MuZhou233) [inline _2021-finalproject/README.md:1]: Your commits sholdn't touch this file

**Review by MuZhou233 — CHANGES_REQUESTED** [review]

File: _2021-finalproject/README.md (hunk 1 above)
Comment (lionLJX) [reply _2021-finalproject/README.md:1]: Is it I should cancel my commit ?

Comment (lionLJX) [reply _2021-finalproject/README.md:1]: but if I cann't touch the file ,where I can make the change to solve the conflict?in the resolved?

Comment (MuZhou233) [reply _2021-finalproject/README.md:1]: No, here shows the finally changes that your commits made, not one single commit.

Comment (lionLJX) [reply _2021-finalproject/README.md:1]: emmm,  Shouldn't I modify it in Git? I am confused now .so  how I can through the check?Cancle my commits of about modify the _2021-finalproject/README.md? But how can I resolve my conflict without modifying the file?
@MuZhou233

Comment (MuZhou233) [reply _2021-finalproject/README.md:1]: You can get inspire from other PR, you are not the only one face to this problem.

**Pull request closed without merging**